test/unit/resources/aws_iam_access_key_test.rb

```diff
@@ -6,27 +6,28 @@
 require 'aws_iam_access_key'
 
 class AwsIamAccessKeyTest < Minitest::Test
-  Username = 'test' 
-  Id = 'id'
-  Date = 'date'
+  Username = 'test'.freeze
+  Id = 'id'.freeze
+  Date = 'date'.freeze
 
   module AccessKeyFactory
     def aws_iam_access_key(decorator = mock_decorator(stub_access_key))
       AwsIamAccessKey.new({ username: Username, id: Id }, decorator)
     end
 
     def stub_access_key(
-      _nil: false,
```

Comment:
No, it was just there for the sake of completeness.  It can go.  Interestingly, I don't see a use of the **create_date** parameter, either.